shared/chat/conversation/messages/wrapper/index.js

```diff
@@ -604,7 +606,7 @@ const styles = Styles.styleSheetCreate({
   timestamp: Styles.platformStyles({
     common: {position: 'relative'},
     isElectron: {top: 1},
-    isMobile: {top: 2},
+    isMobile: {top: 1.5},
```

Comment:
i doubt we want to do this. why is the half pixel necessary

Reply:
I used an on-screen ruler to line it up. It was half a pixel off either way.

Reply:
i think this is indicative of a parent container thats not the right size and then doing a centering

Reply:
Okay, I'll see what I can do to make this better. 👍